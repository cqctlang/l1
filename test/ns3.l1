stab = mktab();
tabinsert(stab, "test", 0x1);
mknsraw(c32le, mktab(), stab);

