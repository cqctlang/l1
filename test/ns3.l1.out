entry: var params, 1 locals, 0 temps
error: invalid raw symbol table
             mknsraw	(builtin function)
               entry	(ns3.l1:3)
